src/mixins/lit-localize-lite-mixin.ts

```diff
@@ -74,6 +80,7 @@ export const litLocalizeLiteMixin = <T extends Constructor<LitElement>>(
       this.resources = {
         [language]: data,
       };
+      this.rtl = await isRTL(this.language!);
```

Comment:
If we would set the `dir` attribute on the parent element, I think we can drop all other code in this PR.

Reply:
The actual <ha-onboarding element is rendered in the HTML template so I'm not sure what to modify there if that's what you mean.
Also the element does not render any wrapper. Add a div?

Reply:
Just use `this.setAttribute("dir", dir);`